include/ssl_key_handler.hpp

```diff
@@ -540,6 +540,13 @@ inline std::optional<boost::asio::ssl::context> getSSLClientContext()
         return std::nullopt;
     }
 
+    // Currently remote server's certificate verfication fails for
+    // HMC provided self-signed certificates,
+    // so skip remote server's certificate verfication.
+
+#if 0 // TODO: certificate verfication can be enabled after supporting
```

Comment:
I never see the #if 0 upstream. Can you just comment this code out? Or the option like @baemyung suggested

Reply:
I also agree to do either - commenting out the code section, or having an option.
It is much better to read the code - especially when we do `blame` or search.

Reply:
done